code/modules/cm_marines/orbital_cannon.dm

```diff
@@ -398,10 +401,40 @@ var/list/ob_type_fuel_requirements
 		return TRUE
 	return FALSE
 
+// proc designed for handling ob camera shakes, takes the target location as input and calculates camera shake based off user location.
+/obj/structure/ob_ammo/warhead/proc/handle_ob_shake(turf/epicenter)
+
+	var/radius_size = 30
+
+	for(var/mob/living/user in urange(radius_size, epicenter))
+
+		var/distance = get_accurate_dist(get_turf(user), epicenter)
+
+		var/distance_percent = ((radius_size - distance) / radius_size)
+
```

Comment:
Must we have all this whitespace? Just group the variables together. Its just kinda weird that almost everything in this proc is double spaced.